packages/governance/src/question.js

```diff
@@ -80,6 +83,15 @@ const looksLikeParamChangeIssue = issue => {
   assertInstance(issue.contract);
 };
 
+/** @type {LooksLikeApiInvocation} */
+const looksLikeApiInvocation = issue => {
```

Comment:
We already have a word for "looks like":
```suggestion
const assertApiInvocation = issue => {
```

I think it would be good to revise the untouched `looksLike` functions too.

Reply:
I think we use "looksLike" when the verification is leaky.  (`looksLikeBrand`)  But I won't say that any of these are leaky or unreliable in the same way, so I've changed them.